### DISCUSSION TITLE
are there prebuilt binaries?

### DISCUSSION THREAD

**Discussion by markmark1** [Uncommon/Xit#373] [Q&A] [answered]
are there prebuilt binaries?

**Comment by Uncommon** [comment, marked as answer]
Yes, they're on the releases page: https://github.com/Uncommon/Xit/releases

**Reply by amuraru** [reply]
@Uncommon is there any plan to provide binaries for newer versions? thanks!

**Reply by Uncommon** [reply]
I'm planning to do a new release soon. I just want to fix a couple of bugs first. You can track progress here: https://github.com/Uncommon/Xit/milestone/19

**Reply by Uncommon** [reply]
[Version 1.0b16](https://github.com/Uncommon/Xit/releases/tag/1.0b16) has been posted.
